### PR TITLE
Update indexer rev to sleep time between request and Hasura metadata loading

### PR DESCRIPTION
**Pull request by musitdev**
# Summary
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.
- **Associated PRs**: https://github.com/movementlabsxyz/aptos-core/pull/66

Define the env var `SLEEP_TIME_BETWENN_REQUEST_MS` to define the sleep time between indexer gRpc request.
By default, it's set to 10 ms.
If `SLEEP_TIME_BETWENN_REQUEST_MS` is et to zero, no sleep is applied between request.

Add a command to load the Hasura metadata.
The meta data file is localed in the folder `networks/suzuka/indexer` and the metadata file is : `hasura_metadata.json`

I've updated the Aptos one.
The file must contain the IP address of the PostgreSQL db so that Hasura can load it. To update dynamically the file with the DB IP and load the metadata, I've created the `load_metadata` binary that can be run this way:
`POSTGRES_DB_HOST=<POSTGRES_HOST_IP> INDEXER_API_URL=<hasura console url> cargo run -p suzuka-indexer-service --bin load_metadata`
 * `POSTGRES_DB_HOST`: set to the IP of the PostgreSQL DB, default `postgres`
 * `INDEXER_API_URL`: Url of the Hasura console, default: `http://127.0.0.1:8085`

# Changelog

- Add SLEEP_TIME_BETWENN_REQUEST_MS to slow indexer request
- Add the `load_metadata`  binary to load Hasura metadata.
# Testing

To test the whole chain locally:
`CELESTIA_LOG_LEVEL=FATAL nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just suzuka-full-node native build.celestia-local.indexer.hasura.indexer-test --keep-tui"`

# Outstanding issues

### CODE REVIEW

**Comment by l-monninger** [comment]
@musitdev Can you make sure this passes before I review?

**Comment by musitdev** [comment]
@l-monninger the test pass now, it made a mistake and the test never end. So I split the 2 in a separate one.
I've updated the PR description for the second time. The description was reverted to the initial one!
This PR contains 2 evolutions: the time between indexer request and the Hasura metadata loading and indexer test update.
You can review the PR.

**Comment by musitdev** [comment]
@l-monninger I made an update and restart the test. I tell you when it's done.

**Comment by musitdev** [comment]
@l-monninger the test has passed.

**Review by radupopa369 — APPROVED** [review]

File: scripts/services/indexer/test_indexer [hunk 1]
```diff
@@ -4,12 +4,14 @@ sleep 10
 result=$(PGPASSWORD=password psql -h $POSTGRES_HOST_IP -U postgres -d postgres -t -c "SELECT COUNT(*) FROM public.transactions;")
 result=$(echo $result | xargs)
 if (( result >= 1 )); then
-  http_status=$(curl -s -o /dev/null -w "%{http_code}" "http://localhost:8085/console")
-  if [ "$http_status" -eq 200 ]; then
+  response=$(curl -s -X POST -H "Content-Type: application/json" -d '{"query":"query {user_transactions { block_height } }"}' "http://localhost:8085/v1/graphql")
```
Comment (l-monninger) [inline scripts/services/indexer/test_indexer:7]: Would be better to move this into a crate at some point.

**Review by l-monninger — APPROVED** [review]
Working for me. But, did you use a dynamic environment to see if this improves those early crashes? Can you link that?

**Comment by musitdev** [comment]
Yes, I test it to synchronize the current suzuka db. The diff is that the suzuka node wasn't processing Tx. It synchronized more than 130M Tx and never crash. I've tested with 10ms and 1ms delay.

File: scripts/services/indexer/test_indexer (hunk 1 above)
Comment (musitdev) [reply scripts/services/indexer/test_indexer:7]: You prefer to implement it in Rust?

**Pull request merged**